packages/fuels/tests/contracts.rs

```diff
@@ -1234,3 +1236,106 @@ async fn multi_call_from_calls_with_different_account_types() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn low_level_call() -> Result<()> {
+    use fuels::types::SizedAsciiString;
+
+    setup_program_test!(
+        Wallets("wallet"),
+        Abigen(
+            Contract(
+                name = "MyCallerContract",
+                project = "packages/fuels/tests/contracts/low_level_caller"
+            ),
+            Contract(
+                name = "MyTargetContract",
+                project = "packages/fuels/tests/contracts/contract_test"
+            ),
+        ),
+        Deploy(
+            name = "caller_contract_instance",
+            contract = "MyCallerContract",
+            wallet = "wallet"
+        ),
+        Deploy(
+            name = "target_contract_instance",
+            contract = "MyTargetContract",
+            wallet = "wallet"
+        ),
+    );
+
+    let function_selector = fn_selector!(initialize_counter(u64));
+    let call_data = calldata!(42u64);
+
+    caller_contract_instance
+        .methods()
+        .call_low_level_call(
+            target_contract_instance.id().clone().into(),
+            Bytes(function_selector),
+            Bytes(call_data),
+            true,
+        )
+        .set_contracts(&[&target_contract_instance])
+        .call()
+        .await?;
+
+    let response = target_contract_instance
+        .methods()
+        .get_counter()
+        .call()
+        .await?;
+    assert_eq!(response.value, 42);
```

Comment:
I would move this assertion to the test in `/examples` and have this test focused on `set_value_multiple_complex`